R/grids.R

```diff
@@ -49,7 +49,10 @@
 grid_regular <- function(x, ..., levels = 3, original = TRUE, filter = NULL) {
   dots <- list(...)
   if (any(names(dots) == "size")) {
-    rlang::warn("`size` is not an argument to `grid_regular()`. Did you mean `levels`?")
+    cli::cli_warn(
+      "{.arg size} is not an argument to {.fn grid_regular}.
+      Did you mean {.arg levels}?"
```

Comment:
could you turn the second sentence into an info bullet via `"i" = "Did you mean.."`?

Reply:
done in https://github.com/tidymodels/dials/pull/350/commits/678cbe5c0d8471358e6b66fc28269a74458b095a